libs/sdk-ui-ext/src/insightView/InsightError.tsx

```diff
@@ -13,21 +13,23 @@ interface IInsightErrorProps {
     error: GoodDataSdkError;
     ErrorComponent?: React.ComponentType<IErrorProps>;
     height?: number | string | null;
+    clientHeight?: number;
 }
 
 const InsightErrorCore: React.FC<IInsightErrorProps & WrappedComponentProps> = ({
     error,
     ErrorComponent = DefaultError,
     height,
     intl,
+    clientHeight,
 }) => {
     const errorMapping = useMemo<IErrorDescriptors>(() => newErrorMapping(intl), [intl]);
     const errorProps = useMemo(() => errorMapping[error.getMessage()] ?? { message: error.message }, [
         errorMapping,
         error,
     ]);
 
-    return <ErrorComponent {...errorProps} height={height} />;
+    return <ErrorComponent {...errorProps} height={height} clientHeight={clientHeight} />;
```

Comment:
what is the difference betweeen height and clientHeight?

Reply:
Height specifies the widget size. If we have custom height selected, we dont want to use this `height` prop since it will override the custom size.